content/workshops/cnd-config-practices.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Configuring a Cloud Native Application"
+draft: false
+description: >
+    You will externalize environment configuration and consume it from
+    a web application on a modern container
+    orchestration platform.
+summary:
+    You will externalize environment configuration and consume it from
+    a web application on a modern container
+    orchestration platform.
+topics:
+tags:
+  - Practices
+  - Development
+  - Spring Boot
+  - Kubernetes
+length: 180
+lab: cnd-config-practices
+---
+
+This is a workshop for developers and application operators
+new to cloud native concepts who wish to learn the basics of
+handling environmental configuration separately from a web application
+codebase using modern orchestration platform tools.
+
+After completing the workshop, you will be able to:
+
+-   Explain configuration requirements of Cloud Native applications
```

Comment:
Will this properly render as a `<ul>`? 

I've never seen a markdown list with so much whitespace between the `-` and the text.

Reply:
@zmb3 Sorry, I almost missed your comment.  

Much of the markdown we authored in the previous master class and product content had liberal whitespace, and line breaks on phrase or sentence separators (for hopefully better diffs between commits).

If TDC has a formal set of maintainer formatting guidelines, if you point me to those, I'll follow for future content edits.